go/internal/doc-snippets/telemetryplugin/exporters.go

```diff
@@ -26,62 +26,131 @@ import (
 	"go.opentelemetry.io/otel/sdk/trace"
```

Comment:
not exaclty sure how these doc snippets work; need to make sure they display well